idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -22,55 +22,59 @@ export default `
       <div class="details-content">
         <div class="details-overview">
           <img src="{{thumbnail}}" width="260">
-          <section>
+          {{#model.measurements}}
```

Comment:
Mainly, I didn't want to copy everything from the model to the view's properties, just the stuff that explicitly needed transformation in the view. That's where I figured it was fine to use the model reference. Ampersand views using the data-hook stuff bind directly to model props usually anyway, except for where the view needs to support some derived properties.